src/sagemaker/amazon/amazon_estimator.py

```diff
@@ -74,10 +74,13 @@ def __init__(
 
         if "enable_network_isolation" in kwargs:
             logger.debug(
-                "removing unused enable_network_isolation argument: %s",
+                "removing unused enable_network_isolation argument in base class: %s",
                 str(kwargs["enable_network_isolation"]),
             )
+            self._enable_network_isolation = kwargs["enable_network_isolation"]
             del kwargs["enable_network_isolation"]
+        else:
+            self._enable_network_isolation = False
```

Comment:
you should be able to eliminate this entire block, and instead make `enable_network_isolation` a named parameter of `__init__` (that will also make it discoverable in the documentation)

Reply:
Done